net/lnet/st/ping.c

```diff
@@ -67,7 +67,7 @@ static void ping_print_qstats(struct nlx_ping_ctx *ctx,
 		"pRECV", "pSEND",
 		"aRECV", "aSEND",
 	};
-	char tbuf[16];
+	char tbuf[64];  //Nalini
```

Comment:
Please remove this.
Or do you want to keep it now, and later remove it when merging to main branch?

Reply:
I have modified because some compiler issue was coming , I will remove my name later. i have tagged it for my tracking my change